graphix/sim/statevec.py

```diff
@@ -264,11 +269,41 @@ def ptrace(self, qargs):
         evals, evecs = np.linalg.eig(rho)  # back to statevector
         self.psi = np.reshape(evecs[:, np.argmax(evals)], (2,) * nqubit_after)
 
+    def remove_qubit(self, qarg):
+        """ Remove a separable qubit from the system and assemble a statevector for remaining qubits.
+        Note this does not exactly perform the partial trace: see :meth:`~graphix.sim.statevec.Statevec.ptrace` and warning therein.
+
+        For a statevector $\ket{\psi} = \sum c_i \ket{i}$ with $i = 00\dots 0 \dots 11\dots 1$, this method returns
```

Comment:
@nabe98 does this sound correct? could you check? 
also could you make sure that docs for this function generates as intended (you'll need to `make html` in `docs/` and check for simulator API doc). I have not checked on my side, I would appreciate if you could debug and check that docs gen work!

Reply:
@shinich1 
I have updated the docs slightliy and see that the intended HTML is generated correctly.

Reply:
thank you!